provider/policy/command.py

```diff
@@ -1,24 +1,28 @@
-from provider.policy.diagram import ProfileDiagram
 from shared.command import BaseCommand, CommandRunner
-from shared.common import *
-from shared.diagram import BaseDiagram
-
-
-class ProfileOptions(BaseOptions):
-    pass
+from shared.common import BaseOptions
+from shared.diagram import NoDiagram, BaseDiagram
 
 
 class Policy(BaseCommand):
+    def __init__(self, region_names, session, diagram):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)